docs/vc-spec.md

```diff
@@ -321,7 +320,9 @@ After receiving the notification that II is ready, the relying party can request
   "method": "request_credential",
   "params": {
     "issuer": {
-        "origin": "https://kyc-resident-info.org"
+        "type": "IC",
```

Comment:
Is this needed?

Reply:
No, I missed that one when removing `type` again. Cood catch!